### DISCUSSION TITLE
Restart?

### DISCUSSION THREAD

**Discussion by StodgyWaif** [mp-se/gravitymon#155] [Q&A]
Is there a way to restart or reboot the device from the web interface?

**Comment by StodgyWaif** [comment]
Okay on a related note 1.3.0 is not detecting my gyro hardware. I rolled back to 1.2.1 to get it working.
I have 5 units built from the same hardware.
If you need any info please let me know and I will try to contribute whatever I can.

**Reply by mp-se** [reply]
A restart function is a good idea, i will add that.

On the 1.3.0 version i have not noticed any issues with gyro but there was some problems with the arduino base that might cause issues. I noticed there was a new version so i can do an new build to see if that works better

**Comment by mp-se** [comment]
You can try out the build in the dev branch, ive added the restart and updated the arduino versions. Let me know it that works with you gyro. If not can you send me the logs from a startup or check the error log
